### PR TITLE
Added 9014f84a-ed8b-42e0-bda9-ac8c969f0d47

### PR DESCRIPTION
**Pull request by d4rk-d4nph3**
I have a small question. I created this TTP from [this](https://unit42.paloaltonetworks.com/unit42-oilrig-targets-middle-eastern-government-adds-evasion-techniques-oopsie/) Unit 42's report.
Where do I have to put this reference link in the yaml file? For example, Sigma rule have a dedicated _reference_ field for this purpose.

### CODE REVIEW

File: ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - OopsIE
+name: Check for virtualization environment
+description: |
+  Malware can use various system checks to detect and avoid virtualization and sandbox environments. 
+  They do this by searching, in results of the system checks, for presence of any artifacts that may indicate that the system is a virtual or sandbox environment.
+  This procedure will use WMI to query information on the physical disk drive and then search for VM artifacts.
+tactic: discovery
+technique:
+  id: T1497.001
+  name: System Checks
```
Comment (khyberspache) [inline ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml:17]: Change this to:
`name: 'Virtualization/Sandbox Evasion: System Checks'`

File: ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+id: 9014f84a-ed8b-42e0-bda9-ac8c969f0d47
+metadata:
+  version: 1
```
Comment (khyberspache) [inline ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml:3]: Delete this, not needed anymore

**Review by khyberspache — CHANGES_REQUESTED** [review]
This is a solid TTP contribution! Just 2 very minor items then I will merge your submission, thanks!

File: ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml (hunk 2 above)
Comment (d4rk-d4nph3) [reply ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml:3]: All three lines?

File: ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml (hunk 1 above)
Comment (d4rk-d4nph3) [reply ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml:17]: I had wondered what to use here. I looked at other TTPs, and found they only mentioned the sub-technique part. 
But, I agree that putting the whole seems to be more informative.

File: ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml (hunk 2 above)
Comment (d4rk-d4nph3) [reply ttps/discovery/9014f84a-ed8b-42e0-bda9-ac8c969f0d47.yml]: Okay.

**Review by Xanthonus — APPROVED** [review]

**Review by khyberspache — APPROVED** [review]

**Pull request merged**